<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QDialog" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main</string>
  </property>
  <widget class="QLabel" name="label_TikTacToe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>46</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tik Tac Toe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Symbol">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Symbol">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Symbol</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CreateProfile">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Stats">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Player1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Player1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Player2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Player2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Play">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProfileError">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_PlayError">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Play Error Label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>mwm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
